tests/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,85 @@
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher-sandbox/rancheros-operator/tests/catalog"
+	"github.com/rancher-sandbox/rancheros-operator/tests/sut"
+)
+
+var (
+	chart      string
+	externalIP string
+)
+
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "os2 e2e test Suite")
+}
+
+var _ = BeforeSuite(func() {
+
+	kubectl := sut.NewKubectl()
+
+	chart = os.Getenv("ROS_CHART")
+	if chart == "" {
+		Fail("No chart provided, skipping tests")
+	}
+	externalIP = os.Getenv("EXTERNAL_IP")
+	if externalIP == "" {
+		Fail("No EXTERNAL_IP provided, skipping tests")
+	}
+
+	By("Deploying ros-operator chart dependencies", func() {
+		err := sut.RunHelmBinaryWithCustomErr("-n", "cert-manager", "install", "--set", "installCRDs=true", "--create-namespace", "cert-manager", "https://charts.jetstack.io/charts/cert-manager-v1.5.3.tgz")
+		Expect(err).ToNot(HaveOccurred())
+
+		err = kubectl.WaitForPod("cert-manager", "app.kubernetes.io/instance=cert-manager", "cert-manager-cainjector")
```

Comment:
:heart_eyes:

Reply:
oh wait, this is ours LOL

I thought we were using an upstream lib to control that :D